app/assets/javascripts/subscription.js

```diff
@@ -0,0 +1,23 @@
+$("#toggler").click(function($event){
+  $event.preventDefault();
+  var link = $(this);
+
+  $.ajax({
+      type: $(this).data("type"),
+      url: $(this).data("path"),
+      success: toggle_subscription(link),
+      error: function (){
+        alert("Something went wrong.")
+      }
+  });
+});
+
+function toggle_subscription(link) {
```

Comment:
AFAIK it is common to use camelCase in JS world (and another JS files in this project are using it also).